### PR TITLE
REL 0.10.4

### PR DESCRIPTION
**Pull request by man-shu**
- Closes none

Changes proposed in this pull request:

- release 0.10.4, with many bug fixes
- highlights to be discussed

### CODE REVIEW

File: doc/changes/0.10.4.rst [hunk 1]
```diff
@@ -5,47 +5,49 @@
 0.10.4.dev
 ==========
```
Comment (Remi-Gau) [inline doc/changes/0.10.4.rst:6]:
```suggestion
0.10.4
======
```

File: doc/changes/0.10.4.rst [hunk 2]
```diff
@@ -5,47 +5,49 @@
...
+----------
+
+- :bdg-dark:`Code` Use the appropriate HTML templates for :class:`nilearn.maskers.MultiNiftiMapsMasker` and :class:`nilearn.maskers.MultiNiftiLabelsMasker` reports that would lead to empty reports for ``MultiNiftiMapsMasker`` (:gh:`4310` by `Rémi Gau`_).
+
 Fixes
 -----
 
 - :bdg-dark:`Code` Fix plotting of carpet plot due to a change in the coming version of matplolib (3.9.0) (:gh:`4279` by `Rémi Gau`_).
 - :bdg-dark:`Code` Fix errant warning when using ``stat_type`` in :func:`nilearn.glm.compute_contrast` (:gh:`4257` by `Eric Larson`_).
 - :bdg-dark:`Code` Fix when thresholding is applied to images by GLM reports (:gh:`4258` by `Rémi Gau`_).
```
Comment (Remi-Gau) [inline doc/changes/0.10.4.rst:20]: I think that we have an extra line between each item of the changelog. Can you check the previous version release? Also this is an implicit thing we should probably mention it somewhere.

Comment (man-shu) [reply doc/changes/0.10.4.rst:20]: Yes indeed! I will add that to the same PR for directives.

File: doc/changes/0.10.4.rst [hunk 3]
```diff
@@ -24,28 +34,44 @@ Fixes
...
 
 - :bdg-success:`API` Add an ``avg_method`` parameter to :func:`nilearn.plotting.plot_surf_stat_map` (:gh:`4298` by `Rémi Gau`_).
 
+- :bdg-primary:`Doc` Advertise nilearn's in built globbing capability in an example (:gh:`4334` by `Himanshu Aggarwal`_).
+
+
 Changes
 -------
 
 - :bdg-dark:`Code` Throw warnings when passing parameters to surface plotting functions that are not used by the plolty engine (:gh:`4298` by `Rémi Gau`_).
```
Comment (Remi-Gau) [inline doc/changes/0.10.4.rst:61]:
```suggestion
- :bdg-dark:`Code` Throw warnings when passing parameters to surface plotting functions that are not used by the plotly engine (:gh:`4298` by `Rémi Gau`_).
```

File: doc/changes/0.10.4.rst [hunk 4]
```diff
@@ -2,19 +2,29 @@
...
-==========
+0.10.4
+======
+
+**Released April 2024**
+
+HIGHLIGHTS
+----------
+
+- :bdg-dark:`Code` Use the appropriate HTML templates for :class:`nilearn.maskers.MultiNiftiMapsMasker` and :class:`nilearn.maskers.MultiNiftiLabelsMasker` reports that would lead to empty reports for ``MultiNiftiMapsMasker`` (:gh:`4310` by `Rémi Gau`_).
```
Comment (Remi-Gau) [inline doc/changes/0.10.4.rst:13]: I think we should just probably say that this is mainly a fix release that fixes bug introduced in the previous release or older bugs (several of them in the nilearn reports)

**Comment by Remi-Gau** [comment]
The numpy2 workflow should pass. So we could proceed with the release.

**Comment by Remi-Gau** [comment]
will fix the merge conflict on this one

**Comment by Remi-Gau** [comment]
@man-shu 
I think this is ready to be merged.
Let's wait for CI to complete but then I would suggest to plan a call tomorrow morning for finalize the release.

**Comment by man-shu** [comment]
> Let's wait for CI to complete but then I would suggest to plan a call tomorrow morning for finalize the release.

Cool! Let's do that.

**Pull request merged**